obs-studio-client/source/controller.cpp

```diff
@@ -316,6 +316,12 @@ void Controller::disconnect()
 {
 	if (m_isServer) {
 		m_connection->call_synchronous_helper("System", "Shutdown", {});
+
+		// Waiting for server to close
+		while (is_process_alive(procId)) {
```

Comment:
We cannot do that here. This is exposing to a potential infinite loop that would freeze the app entirely.